### ISSUE TITLE
matlabbatch for preprocessing

### ISSUE THREAD

**Issue by hanhuili** [wangyirui/AD_Prediction#16]
Dear @wangyirui ,

I am wondering if you can provide the matlabbatch file for image preprocessing in SPM? Thank you in advance

**Comment by wangyirui** [comment]
I can help you check that. Since this batch file is implemented by another teammate, I am not sure if he still keeps this file. I'll let you know if we can find it!

**Comment by hanhuili** [comment]
> I can help you check that. Since this batch file is implemented by another teammate, I am not sure if he still keeps this file. I'll let you know if we can find it!

Thank you very much! By the way, have you tried fine-tuning a 3D CNN without the auto-encoder? How is the result?

**Comment by wangyirui** [comment]
> > I can help you check that. Since this batch file is implemented by another teammate, I am not sure if he still keeps this file. I'll let you know if we can find it!
> 
> Thank you very much! By the way, have you tried fine-tuning a 3D CNN without the auto-encoder? How is the result?

No. Since there is no public large-scale 3D dataset to pre-train a model, training a 3D CNN on a relatively small dataset from scratch is unavoidable (at least the time we implemented our model). But recently, I heard some people say there are some techniques that can perform 3D CNN fine-tuning based on 2D pre-trained model. I suggest you google some related works.

**Comment by THUPZJ** [comment]
Yirui, thanks for your open source kindness, I also need the matlab batch preprocessing files based upon SPM, can you provide me it or just give me some advice, my email is 1581896351@qq.com, in fact, I have also sent a email to you, Thanks! Hope for your respondacne!

**Comment by Hannah-Yao** [comment]
> > I can help you check that. Since this batch file is implemented by another teammate, I am not sure if he still keeps this file. I'll let you know if we can find it!
> 
> Thank you very much! By the way, have you tried fine-tuning a 3D CNN without the auto-encoder? How is the result?

Hi, did you get the files? I am looking for these too. Thanks!

**Comment by Hannah-Yao** [comment]
> Dear @wangyirui ,
> 
> I am wondering if you can provide the matlabbatch file for image preprocessing in SPM? Thank you in advance

Hi @hanhuili, did you get the files? I am looking for these too. Thanks!

**Comment by hanhuili** [comment]
> > Dear @wangyirui ,
> > I am wondering if you can provide the matlabbatch file for image preprocessing in SPM? Thank you in advance
> 
> Hi @hanhuili, did you get the files? I am looking for these too. Thanks!

**Comment by hanhuili** [comment]
> > > Dear @wangyirui ,
> > > I am wondering if you can provide the matlabbatch file for image preprocessing in SPM? Thank you in advance
> > 
> > 
> > Hi @hanhuili, did you get the files? I am looking for these too. Thanks!

No, it is 9012 now and I still haven't received the batch file

**Comment by wangyirui** [comment]
@hanhuili @Hannah-Yao Thanks for your inquiry and sorry for the very late reply. I asked my teammate several months ago, but unfortunately, he cannot find the matlab batch file. At this time, you may need to do it by yourself. This is a final project of Computer Vision class, so the code doesn't organize well due to the limited time. I am planning to polish this project again to make the code more readable. Also, I will use other state-of-art models (AlexNet is too old...). I will include all the code this time. I will do it ASAP in my spare time. Thanks again for your interest in our project.
